### ISSUE TITLE
LiveTV Unable to edit

### ISSUE THREAD

**Issue by RXM307** [roadrunner6/mythcut#3]
Currently Live TV recording are unable to be viewed (nor would you want to edit them) so can these be ignored?

**Comment by roadrunner6** [comment]
Do you mean a recording is shown in mythcut while watching Live-TV with mythtv? If so, never tried that, should be no problem to fix.

**Comment by RXM307** [comment]
Yes LiveTV recordings show up in mythcut along side normal recordings, you cant edit them as no images are shown.

**Comment by RXM307** [comment]
It might be also worth Hiding or atleast Flagging (on the Select movie page) in progress recordings and/or disabling the option to transcode these after cutting as they will transcode however will continue to record on the end of the transcoded recording....
